src/AltinnCore/Designer/Controllers/DeployController.cs

```diff
@@ -115,7 +115,7 @@ public async Task<JsonResult> StartDeployment(string org, string service, string
                         {
                             id = 5,
                         },
-                        parameters = $"{{\"APP_OWNER\":\"{org}\",\"APP_REPO\":\"{service}\",\"APP_DEPLOY_TOKEN\":\"{_sourceControl.GetDeployToken()}\",\"GITEA_ENVIRONMENT\":\"{environment}\", \"APP_COMMIT_ID\":\"{masterBranch.Commit.Id}\"}}\"",
+                        parameters = $"{{\"APP_OWNER\":\"{org}\",\"APP_REPO\":\"{service}\",\"APP_DEPLOY_TOKEN\":\"{_sourceControl.GetDeployToken()}\",\"GITEA_ENVIRONMENT\":\"{environment}\", \"APP_COMMIT_ID\":\"{masterBranch.Commit.Id}\",\"should_deploy\":\"{true}\"}}\"",
```

Comment:
Use capital letters to be consistent with the rest of the parameter names

Reply:
The point with the difference was to indicate that the capitalized parameters are mandetory, while the last one is not